package.json

```
@@ -69,5 +69,10 @@
     "fix": "yarn lint --fix && ./scripts/updateAPIVersion.js",
     "report": "nyc -r text -r lcov report",
     "coveralls": "cat coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js"
+  },
+  "exports": {
+    "browser": "./lib/stripe.worker.js",
+    "worker": "./lib/stripe.worker.js",
+    "default": "./lib/stripe.js"
```

Alternatively, we can have this be:
```suggestion
    "node": "./lib/stripe.js",
    "default": "./lib/stripe.worker.js" // stripe.default.js ?
```
This is [what is suggested in the Node docs](https://nodejs.org/api/packages.html#conditional-exports:~:text=For%20this%20reason%2C%20using%20%22node%22%20and%20%22default%22%20condition%20branches%20is%20usually%20preferable%20to%20using%20%22node%22%20and%20%22browser%22%20condition%20branches.), but I'd prefer to test the waters with just "worker" and "browser" environments first.

---

We should keep the same default as we have now.